mutiny-core/src/ldkstorage.rs

```diff
@@ -84,7 +84,13 @@ impl MutinyNodePersister {
     // that has the concatenated node_id
     fn read_value(&self, _key: &str) -> Result<Vec<u8>, MutinyError> {
         let key = self.get_key(_key);
-        self.storage.get(key).map_err(MutinyError::read_err)
```

Comment:
Did this return an error before if the value did not exist?

Reply:
Yeah